lib/core/scripts.rb

```diff
@@ -1,34 +0,0 @@
-# frozen_string_literal: true
-
-module Scripts
-  module_function
-
-  def assert_replicas(gvc:, workload:, location:)
-    <<~SHELL
-      REPLICAS_QTY=$( \
-        curl ${CPLN_ENDPOINT}/org/shakacode-staging/gvc/#{gvc}/workload/#{workload}/deployment/#{location} \
-        -H "Authorization: ${CONTROLPLANE_TOKEN}" -s | grep -o '"replicas":[0-9]*' | grep -o '[0-9]*')
-
-      if [ "$REPLICAS_QTY" -gt 0 ]; then
-        echo "-- MULTIPLE REPLICAS ATTEMPT: $REPLICAS_QTY --"
-        exit -1
-      fi
-    SHELL
-  end
-
-  def helpers_cleanup
-    <<~SHELL
-      unset CONTROLPLANE_RUNNER
-    SHELL
-  end
-
-  # NOTE: please escape all '/' as '//' (as it is ruby interpolation here as well)
-  def http_dummy_server_ruby
```

Comment:
This was the only method that was being used from this module, and I don't understand why we can't just use `tail -f /dev/null`, so I did that in `run` and dropped this.